stdlib/public/Concurrency/GlobalExecutor.cpp

```diff
@@ -167,3 +176,25 @@ void swift::swift_task_enqueueGlobal(Job *job) {
   dispatch_async_f(queue, dispatchContext, dispatchFunction);
 #endif
 }
+
+
+/// Enqueues a task on the main executor.
+/// FIXME: only exists for the quick-and-dirty @MainActor implementation.
+void swift::swift_task_enqueueMainExecutor(Job *job) {
+  assert(job && "no job provided");
+
+#if SWIFT_CONCURRENCY_COOPERATIVE_GLOBAL_EXECUTOR
+  insertIntoJobQueue(job);
```

Comment:
If dispatch isn't available, we'll hit this bit of incorrect code. Should I get it working in this case too?